### PR TITLE
Add gauge weight vote results for week 1

### PR DESCRIPTION
**Pull request by penandlim**
Set the weights determined by https://snapshot.org/#/saddlefinance.eth/proposal/0x81b6204967d383f035f570cea9c43dca9b51910a23fcc19638df00b4260a7a6d

```
FraxBP       0xB2Ac3382dA625eb41Fc803b57743f941a484e2a6: 7786
SLP          0xc64F8A9fe7BabecA66D3997C9d15558BF4817bE3: 1038
FraxBP-sUSD  0x104F44551386d603217450822443456229F73aE4: 447
FraxBP-alUSD 0x953693DCB2E9DDC0c1398C1b540b81b63ceA5e16: 447
Frax3pool    0x13Ba45c2B686c6db7C2E28BD3a9E8EDd24B894eD: 94
D4           0x702c1b8Ec3A77009D5898e18DA8F8959B6dF2093: 90
FraxBP-USDT  0x6EC5DD7D8E396973588f0dEFD79dCA04F844d57C: 89
WRen-sBTC    0x17Bde8EBf1E9FDA85b9Bd1a104266b394E9Db33e: 6
WCUSD        0x3dC88ee38db8C7b6DCEB447E4348e51bd87ced93: 2
USDv2        0x7B2025Bf8c5ee8Baad9da8C3E3Ee45E96ed8b8EA: 0
sUSD V3      0x2683190e31e8ce47467c98ff1DBc018aCDD43C2f: 0
alETH        0x8B701e9B3a1887fE9b0C7936a8233b39408e69f6: 0
tBTC         0xB79B4fCF7cB4A1c4064Ff5b48F71A331880ab53a: 0
```

### CODE REVIEW

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")  # prompts for password
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    gauge_controller = multisig.contract(
+        GAUGE_CONTROLLER_ADDRESS[CHAIN_IDS["MAINNET"]])
+
+    gauge_to_relative_weight_dict = {
+        "0xB2Ac3382dA625eb41Fc803b57743f941a484e2a6": 7786,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:19]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        "0xc64F8A9fe7BabecA66D3997C9d15558BF4817bE3": 1038,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:20]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        "0x104F44551386d603217450822443456229F73aE4": 447,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:21]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        "0x953693DCB2E9DDC0c1398C1b540b81b63ceA5e16": 447,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:22]: ✅  new alUSD gauge

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        "0x13Ba45c2B686c6db7C2E28BD3a9E8EDd24B894eD": 94,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:23]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        "0x702c1b8Ec3A77009D5898e18DA8F8959B6dF2093": 90,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:24]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+        "0x6EC5DD7D8E396973588f0dEFD79dCA04F844d57C": 89,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:25]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+        "0x17Bde8EBf1E9FDA85b9Bd1a104266b394E9Db33e": 6,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:26]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+        "0x3dC88ee38db8C7b6DCEB447E4348e51bd87ced93": 2,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:27]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+        "0x7B2025Bf8c5ee8Baad9da8C3E3Ee45E96ed8b8EA": 0,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:28]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+        "0x2683190e31e8ce47467c98ff1DBc018aCDD43C2f": 0,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:29]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+        "0x8B701e9B3a1887fE9b0C7936a8233b39408e69f6": 0,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:30]: ✅

File: scripts/2022_07_03_set_votes_for_week_1.py [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+        "0xB79B4fCF7cB4A1c4064Ff5b48F71A331880ab53a": 0,
```
Comment (hammeiam) [inline scripts/2022_07_03_set_votes_for_week_1.py:31]: ✅

**Review by hammeiam — APPROVED** [review]

**Review by hannespfeiffer — APPROVED** [review]

**Pull request merged**